src/main/resources/assets/thaumicboots/lang/en_US.lang

```diff
@@ -87,6 +91,7 @@ ExplorationsEMT.1=As you continued your studies of electricity, you noticed some
 SeasonalCore.1=In your studies of magickal phenomena, you discovered something peculiar: there are spikes of magickal energies around specific parts of the solar calendar. <BR><BR>These spikes have been known to cause various tranformations, such as transforming chests to presents, and vice versa as they pass. Interestingly, you've managed to capture this chaotic energy. <BR><BR>Yet you wonder if there is some way you may be able to stabilize these energies...
 SeasonalStabilized.1=As you continued your studies of the seasonal magick phenomena, you noticed something interesting, it actually is possible to attune these energies into a specific phenomena, resulting in a drastically boosted seasonal effect... with one caveat: these effects become incredibly diminished after the season has passed. <BR><BR>The first of these effects you've noticed is a drastic increase in the steady running speed over time, while freezing water similar to that of comet attuned magicks <BR><BR>Though, you almost swear you can hear sleigh bells...
 SeasonalStabilized.2=You believe you should come back to this in December...
+UniqueCore.1=In your studies of magickal phenomena, you've run upon something peculiar: there are certain magicks that can only be recreated in exact scenarios. The first of these you've stumbled unto being a seemingly simple slowness curse. <BR><BR>However, while you were able to reproduce the boots themselves, you cannot for the life of you understand how the magick energy works, nor how to recreate it directly. <BR><BR>You begin to wonder if you'll stumble upon energies in similar situations...
```

Comment:
> unto

Do you mean "onto" or "upon" here?
Unto means to or until.

Reply:
Op, yeah meant onto

Reply:
Gonna use upon tho, sounds way better